views/eApostilles/openEApp.ejs

```diff
@@ -150,5 +150,5 @@
 
         <a href="/download-receipt/<%= applicationId %>" class="govuk-link govuk-!-margin-top-2" role="button" download="<%= applicationId %>.pdf">Download your receipt</a>
 
-        <a href="/test-download" class="govuk-link govuk-!-margin-top-2" role="button" download="test.png">Test download</a>
+        <a href="/test-download" class="govuk-link govuk-!-margin-top-2" role="button">Test download</a>
```

Comment:
what's test download for? is it meant to be merged?

Reply:
ah, so this is a completely separate issue not related to this PR. Basically, we have an issue (which you probably know about) where if a request takes longer than 5 seconds it tries again up to 3 times. In order to test this in AWS I would usually create the code on a new branch and promote that branch to be deployed via Teraform. 

However this time round there were loads of changes made to the Terraform files that would have taken too long to try and fix, so I applied the changes directly onto the `demo` branch with the intention of removing it at the end of the sprint (so tomorrow).

The change you're commenting on was one I accidentally made on the wrong branch so it actually shouldn't be in this pr :)